api.swagger.yaml

```diff
@@ -2665,6 +2680,23 @@ components:
           description: Reason for flagging post as inappropriate
           type: string
           example: I find this offensive
+    post-category-schema:
+      properties:
+        name: 
+          description: 'Category name'
+          type: string
+          example: Note
+        uuid:
+          description: 'This is the Invotra UUID of the parent job role, optional.'
+          pattern: >-
+            ^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$
+          example: 01234567-89ab-cdef-1234-56789abcdef0
+          type: string
+        category_colour:
+          description: Hexidecimal colour for the category
+          type: string
+          example: FFF
```

Comment:
Let's use the same 'CSS colours' language as we did before, for consistency and also clarity (as 3 hex digits wouldn't be sufficient to represent all our current colours anyway)